circuit/src/scaffold.ts

```diff
@@ -23,7 +20,7 @@ export abstract class AxiomBaseCircuitScaffold<T> extends BaseCircuitScaffold {
   protected halo2Lib!: Halo2LibWasm;
```

Comment:
resultLen is already in computeQuery which is in both queryBuilder and this.computeQuery. I guess there is some ordering about when each one becomes defined. but if possible, try to remove redundancy.

Reply:
removed queryBuilderBase; will keep this.resultLen because it is used to set the value this.computeQuery